src/app/beer_garden/api/http/__init__.py

```diff
@@ -44,25 +45,35 @@
 client_ssl = None
 
 
-def run():
+def signal_handler(signal_number, stack_frame):
```

Comment:
Ok. I understand what is going on now.

Reply:
Yeah. We could maybe eventually switch out Tornado for something else, but until we do we're pretty much tied to doing things this way since mixing asyncio Tornado and threaded Everything Else in the same process would be messy.